types/custom-functions-runtime/index.d.ts

```diff
@@ -75,6 +75,15 @@ declare namespace CustomFunctions {
          * [Api set: CustomFunctionsRuntime 1.3]
          */
         parameterAddresses?: string[];
+
+        /**
+         * This flag determines whether the function is being invoked as part of Formula Value Preview execution or not.
```

Comment:
It looks like we don't use "Formula Value Preview" for customer-facing documentation. Could we say this? 
```suggestion
         * Determines whether the function is invoked as part of the formula value preview.
```